updatecli/updatecli.d/plugin-site-api.yaml

```diff
@@ -14,12 +14,13 @@ scms:
 
 sources:
   latestRelease:
-    kind: dockerimage
-    name: "Get latest jenkinsciinfra/plugin-site-api version"
+    kind: githubrelease
+    name: Get jenkins-infra/incrementals-publish latest version
     spec:
-      image: "jenkinsciinfra/plugin-site-api"
-      architecture: "amd64"
-      tagfilter: "^\\d+-[a-z0-9]+$"
+      owner: "jenkins-infra"
+      repository: "plugin-site-api"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
 
 targets:
```

Comment:
You have to add a condition to check if the Docker image is present, otherwise you'll have PR with failing checks until the releases are complete